src/main/java/software/amazon/sns/AmazonSNSExtendedClient.java

```diff
@@ -19,6 +19,7 @@
 
 public class AmazonSNSExtendedClient extends AmazonSNSExtendedClientBase {
     static final String MULTIPLE_PROTOCOL_MESSAGE_STRUCTURE = "json";
+    private static final String S3_KEY = "S3Key";
```

Comment:
This should be put near line 26 as they are both `private static final`

Reply:
changed.